websites/O/OK/iframe.ts

```diff
@@ -0,0 +1,14 @@
+const iframe = new iFrame();
+
+iframe.on("UpdateData", async () => {
+	if (document.querySelector("video")) {
```

Comment:
This conditional is not needed - if the element doesn't exist `video` will be `false` and be falsey at the duration check.